Src/WitsmlExplorer.Frontend/components/Modals/ConfirmModal.tsx

```diff
@@ -19,6 +20,7 @@ const ConfirmModal = (props: ConfirmProps): React.ReactElement => {
       onSubmit={props.onConfirm}
       isLoading={false}
       confirmText={props.confirmText ?? "Yes"}
+      cancelText={props.cancelText ?? "No"}
```

Comment:
Is it necessary to set `?? "No"`? If no cancelText is specified, I think we should stick to ModalDialog's default cancel text.